lab-ron/test/greet-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const greet = require(__dirname + '/../lib/greetings');
+const assert = require('assert');
+
+describe('testing module greetings', function(){
+  describe('testing function', function(){
+    it('should return the greeting + the argument "Ron"', function(){
+      process.argv[2] = 'Ron';
+      assert.equal(greet(process.argv[2]), 'hello Ron');
+      // assert.equal(greet('Yunjoo'), 'hello Yunjoo');
```

Comment:
Just to note again, do no have any commented out code in a PR, unless it is absolutely necessary.